euth/communitydebate/templates/euth_communitydebate/topic_form.html

```diff
@@ -67,6 +68,56 @@ <h3 class="sans-serif">
                 {{ form.image.errors }}
             </div>
 
+    {% with upload_forms.max_num as fileupload_max %}
+    <h3>{% trans 'Downloads' %}</h3>
+    {% getAllowedFileTypes as allowed_file_types %}
+    <p>{% blocktrans %} You can upload a maximum of {{ fileupload_max }} documents. Allowed formats are {{ allowed_file_types }}. {% endblocktrans %}</p>
+    {% for form in upload_forms %}
+    <div class="card card-body">
+        {% for field in form.visible_fields %}
+        {% if field.name == 'title' or field.name == 'document' %}
```

Comment:
Otherwise the topic would show, right? Maybe we can add the topic as hidden field and by that circumvent the other topic=none thing in the view? But I will check that out!